test/utils/computeLedgerHash.ts

```diff
@@ -4,31 +4,42 @@ import { ValidationError } from '../../src/common/errors'
 import { computeLedgerHeaderHash } from '../../src/utils'
 import requests from '../fixtures/requests'
 import responses from '../fixtures/responses'
-import { assertResultMatch } from '../testUtils'
 
 const { computeLedgerHash: REQUEST_FIXTURES } = requests
 
-describe('computeLedgerHash', function () {
+describe('Compute Ledger Hash', function () {
```

Comment:
IMO this should stay as the function name, rather than translating it into English